packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -334,6 +335,8 @@ export interface CallCompositeStrings {
     copyInviteLinkActionedAriaLabel: string;
     copyInviteLinkButtonLabel: string;
     defaultPlaceHolder: string;
+    deniedPermissionToRoomDetails?: string;
```

Comment:
Are these strings added to the stable API as well? 

I don't see the stable API file at all. Did you build for stable flavor as well?